packages/flame/lib/src/geometry/rectangle_component.dart

```diff
@@ -58,6 +60,7 @@ class RectangleComponent extends PolygonComponent {
     super.paint,
     super.paintLayers,
     super.shrinkToBounds,
+    super.scale,
```

Comment:
scale already exists in this constructor, I must have accidentally only added it to one
```suggestion
```